### PR TITLE
Add to individual contributor

### PR DESCRIPTION
**Pull request by fxue**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/93)

### CODE REVIEW

File: members.yaml [hunk 1]
```diff
@@ -73,3 +73,9 @@
   lastName: Benton
   affiliation: Red Hat
   email: willb@redhat.com
+
+- github: fxue
```
Comment (gaocegege) [inline members.yaml]: Please keep this file alphabetized by github id.

**Comment by ankushagarwal** [comment]
@fxue : Can you rebase?

**Comment by texasmichelle** [comment]
Ping @fxue to rebase & alphabetize

**Comment by texasmichelle** [comment]
@fxue how are you not in this file yet? Rebase :)

**Comment by abhi-g** [comment]
Closed because stale..

**Pull request closed without merging**